lib/Sema/TypeCheckPattern.cpp

```diff
@@ -64,13 +64,21 @@ extractEnumElement(TypeChecker &TC, DeclContext *DC, SourceLoc UseLoc,
 ///
 /// If there are no enum elements but there are properties, attempts to map
 /// an arbitrary property to an enum element using extractEnumElement.
+///
+/// \param isPromoted If set to anything but the \c nullptr, this will be set to
```

Comment:
Again, would be best to give this an explicit name. But since this is transitional, maybe not worth it...